api/src/test/java/org/pmiops/workbench/db/dao/CohortDaoTest.java

```diff
@@ -43,7 +54,7 @@ public void setUp() {
     dbWorkspace.setLastModifiedTime(timestamp);
     workspaceDao.save(dbWorkspace);
 
-    dbUser = userDao.save(ReportingTestUtils.createDbUser());
+    dbUser = userDao.save(userFixture.createEntity());
```

Comment:
I shied away from pulling all the test fixture code out of ReportingTestUtils for this branch, but there's support in the works on the Ruby side to make this much cleaner.